src/BloomExe/Edit/AudioRecording.cs

```diff
@@ -236,8 +236,22 @@ public void BeginMonitoring()
 			}
 			if (RecordingDevice != null)
 			{
-				Recorder.BeginMonitoring();
-				_monitoringAudio = true;
+				try
+				{
+					Recorder.BeginMonitoring(catchAndReportExceptions: false);
+					_monitoringAudio = true;
+				}
+				catch (NAudio.MmException)
+				{
+					var msg = LocalizationManager.GetString("EditTab.Toolbox.TalkingBookTool.MicrophoneAccessProblem",
+						"Bloom was not able to access a microphone.");
+					_webSocketServer.SendString(kWebsocketContext, "micError", msg);
+				}
+				catch (Exception e)
+				{
+					ErrorReport.NotifyUserOfProblem(new ShowOncePerSessionBasedOnExactMessagePolicy(),
```

Comment:
I did this to retain what the original behavior would be if a different exception is throw. But not sure what we want the behavior to be here